<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwidget</class>
 <widget class="QWidget" name="loginwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="LoginTitle">
     <property name="styleSheet">
      <string notr="true">#LoginTitle
{
	color:  black;
	background-color: transparent;
	border: none;
	font: 16pt;
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="LoginMsg">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#LoginMsg
{
	color:  black;
	background-color: transparent;
	border: none;
	font: 12pt;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LoginName">
     <property name="styleSheet">
      <string notr="true">#LoginName
{
	border: none;
    color: #565656;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LoginPassword">
     <property name="styleSheet">
      <string notr="true">#LoginPassword
{
	border: none;
    color: #565656;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="LoginButtons" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="CleanButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#CleanButton
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ConfirmButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#ConfirmButton
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="LoginPasswordForgot">
     <property name="styleSheet">
      <string notr="true">#LoginPasswordForgot
{
	background-color: transparent;
	border: none;
	font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Esqueci Minha Senha</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="LoginNameForgot">
     <property name="styleSheet">
      <string notr="true">#LoginNameForgot
{
	background-color: transparent;
	border: none;
	font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Esqueci Meu Usuario</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
